### ISSUE TITLE
inputProps() returning `required` even for `optional()` schema

### ISSUE THREAD

**Issue by kiliman** [kiliman/remix-params-helper#22]
### Describe the bug

inputProps() returning `required` even for `optional()` schema

### Your Example Website or App

n/a

### Steps to Reproduce the Bug or Issue

n/a

### Expected behavior

n/a

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by WesleyKapow** [comment]
Just hit this.

**Comment by kiliman** [comment]
Yeah. I’m reworking the whole client side part of the helper. Sorry about that. I hope to have this fixed soon.
